bin/webpack.js

```diff
@@ -1,45 +1,26 @@
-#!/usr/bin/env node
-
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
 var path = require('path');
-// var fs = require('fs');
-// Local version replace global one
-
-process.title = 'webpack';
 
+// Local version replace global one
 try {
-	var localWebpack = require.resolve(path.join(process.cwd(), 'node_modules', 'webpack-cli', 'bin', 'webpack.js'));
-	if(localWebpack && path.relative(localWebpack, __filename) !== '') {
+	var localWebpack = require.resolve(path.join(process.cwd(), 'node_modules',
+	'webpack', 'node_modules', 'webpack-cli', 'bin', 'webpack.js'));
+	if(__filename !== localWebpack) {
 		return require(localWebpack);
-	}
-} catch(e) {
-	/*
-	// Delete Cache, Retry and leave it if the path isn`t valid.
-	// swap with webpack-cli later
-	var dirPath = path.join(process.cwd(), 'node_modules', 'webpack', 'bin');
-	var retryLocal;
-	fs.readdir(dirPath, function(err, files) {
-		files.filter( (file) => {
-			delete require.cache[file];
-		});
-
-		// TODO: If path is wrong, leave it to higher powers until we publish on npm
-		// Could also install and load and install through github
-		return require(path.join(dirPath, 'webpack.js'))
-	});
-	*/
-}
+	} //eslint-disable-next-line
+} catch(e) {}
```

Comment:
I did a try on this earlier, figure its best to push a commit for this after we've got it linked with the webpack repo to make sure the catch case is working ( see how they do it in above block )